### PR TITLE
add add-cluster-role-to-user support for -z

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin/issues/12791

cc @smarterclayton @enj

### CODE REVIEW

File: pkg/cmd/admin/policy/modify_roles.go [hunk 1]
```diff
@@ -252,10 +256,12 @@ func NewCmdRemoveClusterRoleFromUser(name, fullName string, f *clientcmd.Factory
 		},
 	}
 
+	cmd.Flags().StringSliceVarP(&saNames, "serviceaccount", "z", saNames, "service account in the current cluster to use as a user")
```
Comment (smarterclayton) [inline pkg/cmd/admin/policy/modify_roles.go]: in the current namespace

File: pkg/cmd/admin/policy/modify_roles.go [hunk 2]
```diff
@@ -204,6 +205,8 @@ func NewCmdAddClusterRoleToUser(name, fullName string, f *clientcmd.Factory, out
...
```
Comment (smarterclayton) [inline pkg/cmd/admin/policy/modify_roles.go]: in the current namespace

**Comment by juanvallejo** [comment]
[test]

**Comment by juanvallejo** [comment]
check_future flaked on https://github.com/openshift/origin/issues/12911 re[test]

**Comment by juanvallejo** [comment]
@smarterclayton or @enj are there any more comments for this patch? Thanks!

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**